### PR TITLE
CJCollins - Plano information file

### PR DESCRIPTION
**Pull request by cjcollins**

### CODE REVIEW

File: CjCollins-Plano.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Plano Tx
+
+## Places to Eat
+- Cozy Thai
```
Comment (chianc1) [inline CjCollins-Plano.md]: I like Thai food.

Comment (githubteacher) [inline CjCollins-Plano.md]: Huge fan of Pad Thai, um, how spicy do they get? Recommendation on 🌶 level?

File: CjCollins-Plano.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+- Iravat India�s Bistro
+- Lazy Dog
```
Comment (githubteacher) [inline CjCollins-Plano.md]: Any recommendations for something to get from Lazy Dog?

File: CjCollins-Plano.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# Plano Tx
```
Comment (githubteacher) [inline CjCollins-Plano.md:1]: Any recommendations on places to visit in Plano, Tx?

**Review by githubteacher — COMMENTED** [review]
Couple things to address before its ready to 🚢

File: CjCollins-Plano.md (hunk 1 above)
Comment (cjcollins) [reply CjCollins-Plano.md]: If you like it really hot, ask for Thai spicy :) Medium spicy is almost too much for most ;)

Comment (cjcollins) [reply CjCollins-Plano.md]: Me too!

**Review by githubteacher — APPROVED** [review]
This looks great!

**Pull request merged**